tailcfg/tailcfg.go

```diff
@@ -665,6 +665,10 @@ type MapResponse struct {
 	// forms are coming later.
 	Domain string
 
+	// CollectServices reports whether this node's Tailnet has
+	// requested that info about services be included in HostInfo.
+	CollectServices bool `json:",omitempty"`
```

Comment:
You might want to make this a `*bool` or `opt.Bool` so we don't need to waste bandwidth re-sending this on every MapResponse when the policy hasn't changed. (most the other fields have been moving towards being only-send-on-changes)